sdk/core/Azure.Core.TestFramework/src/TestEnvironment.cs

```diff
@@ -303,26 +308,7 @@ private async Task ExtendResourceGroupExpirationAsync()
                 return;
             }
 
-            string tenantId = GetOptionalVariable("TENANT_ID");
-            string clientId = GetOptionalVariable("CLIENT_ID");
-            string clientSecret = GetOptionalVariable("CLIENT_SECRET");
-            string authorityHost = GetOptionalVariable("AZURE_AUTHORITY_HOST");
-
-            if (tenantId == null || clientId == null || clientSecret == null || authorityHost == null || ResourceManagerUrl == null)
-            {
-                return;
-            }
-
-            // intentionally not using the Credential property as we don't want to throw if the env vars are not available, and we want to allow this to vary per environment.
```

Comment:
I don't think this made sense because GetRecordedVariable will attempt to bootstrap if the env var is not set.